app/(drawer)/settings.tsx

```diff
@@ -0,0 +1,195 @@
+
+  const onDirectionHandledWithoutMovement = useCallback(
+    (movement: Direction) => {
+      console.log("Direction " + movement);
+      if (movement === 'left' && focusedIndex === 0) {
+        navigation.dispatch(DrawerActions.openDrawer());
+        toggleMenu(true);
+      }
+    },
+    [toggleMenu, focusedIndex, navigation],
+  );
+
+  const menuItems = [
+    { id: 'language', label: 'Language' },
+    { id: 'sound', label: 'Sound' },
+    { id: 'display', label: 'Display' },
+  ];
+
+  const submenus = {
+    language: ['English', 'French', 'Spanish'],
+  };
+  const [selectedMenuItem, setSelectedMenuItem] = useState('language'); 
+  const [focusedMenuItem, setFocusedMenuItem] = useState('language'); 
+  const [selectedSubmenuItem, setSelectedSubmenuItem] = useState("English");
+
+  return (
+    <SpatialNavigationRoot isActive={isActive}
+      onDirectionHandledWithoutMovement={onDirectionHandledWithoutMovement}>
+      <Stack.Screen options={{ headerShown: false }} />
+      <Text style={styles.heading}>Account</Text>
+      <View style={styles.container}>
+        
+      <DefaultFocus>
+      <SpatialNavigationFocusableView>
+      <View style={styles.container}>
+       
+      {/* Main Menu (Left) */}
+      <View style={styles.mainMenu}>
+        {menuItems.map((item) => (
+          <TouchableOpacity
+            key={item.id}
+            style={[
+              styles.menuItem,
+              focusedMenuItem === item.id && styles.focusedMenuItem,
+              selectedMenuItem === item.id && styles.selectedMenu,
+            ]}
+            onPress={() => setSelectedMenuItem(item.id)} 
+            onFocus={() => setFocusedMenuItem(item.id)} 
+            onBlur={() => setFocusedMenuItem(null)} 
+          >
+            <Text style={[styles.menuText, selectedMenuItem === item.id && styles.selectedMenuItem]}>{item.label}</Text>
+          </TouchableOpacity>
+        ))}
+      </View>
+
+      {/* Submenu (Right) */}
+     {submenus[selectedMenuItem] && <View style={styles.submenu}>
+      <Text style={styles.title}>Settings</Text>
+        {selectedMenuItem && submenus[selectedMenuItem]?.map((option, index) => (
+          <TouchableOpacity
+            key={index}
+            style={[styles.submenuItem, selectedSubmenuItem === option && styles.focusedSubMenuItem]}
+            onPress={() => console.log(`Selected: ${option}`)} 
+          >
+            <Text style={[styles.submenuText, selectedSubmenuItem === option && styles.selectedSubmenuText]}>{option}</Text>
+          {selectedSubmenuItem === option &&
+              <Icon name="check" size={20} color="#00FF00" style={[styles.checkIcon, selectedSubmenuItem === option && styles.selectedCheckIcon]} />
+          }
+          </TouchableOpacity>
+        ))}
+      </View>
+}
+    </View>
+      </SpatialNavigationFocusableView>
+      </DefaultFocus>
+      </View>
+    </SpatialNavigationRoot>
+  );
+}
+
+const useTVStyles = function() {
```

Comment:
nitpick: naming convention should be more align towards the setttings rather that "tvstyles"